otlp/package.yaml

```diff
@@ -8,6 +8,7 @@ version:             0.0.1.0
 extra-source-files:
 - README.md
 - ChangeLog.md
+- proto/**/*/*.proto
```

Comment:
Matches all `.proto` files in the `proto` submodule. The `proto` submodule was moved from the root directory to `otlp` because `hs-opentelemetry-otlp` needs it to auto-generate `.hs` files and no other packages use it.